src/sass/_mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin image-dimensions {
```

Comment:
I would consider changing the name to more meaningful - answering to question "what this mixin does?". So `fullSize`, `fill` or etc. would be much better IMHO.